### PR TITLE
Adjust index column width

### PR DESCRIPTION
**Pull request by barmac**
__Which issue does this PR address?__

This PR is created to mitigate a problem introduced in https://github.com/bpmn-io/dmn-js/pull/530

drag

Notice how drag rule icon is moved on top of index number.

__Definition of Done__

* [x] corresponds to [the design principles](https://github.com/bpmn-io/design-principles)
* [x] corresponds to [the code standards](https://github.com/bpmn-io/bpmn-js/blob/master/.github/CONTRIBUTING.md#creating-a-pull-request)
* [x] passes Continuous Integration checks
* [x] available as feature branch on GitHub
  * [x] contains the cleaned up commit history
  * [x] commit messages satisfy our [commit message guidelines](https://www.conventionalcommits.org/)
  * [ ] a single commit closes the issue via Closes #issuenr

This PR includes https://github.com/bpmn-io/dmn-js/pull/530

### CODE REVIEW

**Comment by barmac** [comment]
Note for reviewer: If you want, I can add the commit to #530 and just close this PR. I didn't want to add complexity to already complex PR, though.

**Comment by barmac** [comment]
Rebased on top of `develop`.

**Comment by oguzeroglu** [comment]
Reviewing this now.

File: packages/dmn-js-decision-table/src/features/decision-table-head/components/DecisionTableHead.js [hunk 1]
```diff
@@ -42,6 +42,15 @@ export default class DecisionTableHead extends Component {
     this._changeSupport.offElementsChanged(root.id, this.onElementsChanged);
   }
 
+  getIndexColumnWidth() {
+    const root = this._sheet.getRoot(),
+          rowsCount = root.rows.length;
+
+    const width = 40 + Math.floor(Math.log10(rowsCount)) * 8;
```
Comment (oguzeroglu) [inline packages/dmn-js-decision-table/src/features/decision-table-head/components/DecisionTableHead.js:49]: 🧑‍🔬

Comment (oguzeroglu) [reply packages/dmn-js-decision-table/src/features/decision-table-head/components/DecisionTableHead.js:49]: So the idea here is to give some defined amount of pixels per each digit of the index number right?

Comment (barmac) [reply packages/dmn-js-decision-table/src/features/decision-table-head/components/DecisionTableHead.js:49]: Exactly.

Comment (barmac) [reply packages/dmn-js-decision-table/src/features/decision-table-head/components/DecisionTableHead.js:49]: 1 -> 40px
10 -> 48px
100 -> 56px and so forth

Comment (barmac) [reply packages/dmn-js-decision-table/src/features/decision-table-head/components/DecisionTableHead.js:49]: I will add a comment with short explanation.

Comment (oguzeroglu) [reply packages/dmn-js-decision-table/src/features/decision-table-head/components/DecisionTableHead.js:49]: Yeah makes sense probably, it took me a while to get what's going on.

Comment (barmac) [reply packages/dmn-js-decision-table/src/features/decision-table-head/components/DecisionTableHead.js:50]: OK done.

Comment (philippfromme) [reply packages/dmn-js-decision-table/src/features/decision-table-head/components/DecisionTableHead.js:50]: How do we know a digit is always 8px?

Comment (barmac) [reply packages/dmn-js-decision-table/src/features/decision-table-head/components/DecisionTableHead.js:50]: It's a rough estimation based on double-digit and triple-digit numbers' width.

Comment (oguzeroglu) [reply packages/dmn-js-decision-table/src/features/decision-table-head/components/DecisionTableHead.js:50]: Should we maybe programmatically calcualte the total size? I'm not sure if this is likely but this approach may create problems if we end up changing the CSS in the future.

Comment (oguzeroglu) [reply packages/dmn-js-decision-table/src/features/decision-table-head/components/DecisionTableHead.js:50]: This may be an overkill too though.

Comment (barmac) [reply packages/dmn-js-decision-table/src/features/decision-table-head/components/DecisionTableHead.js:50]: TBH I am afraid that could complicate the code + cause reflow. I haven't tried it, though. I am happy with this rough solution as there are not many cases to handle.

Comment (oguzeroglu) [reply packages/dmn-js-decision-table/src/features/decision-table-head/components/DecisionTableHead.js:50]: Personally I'm ok with the current solution too.

**Comment by barmac** [comment]
@philippfromme suggested to just change the width to accomodate for triple-digit numbers. This is how it looks like:

image

**Comment by barmac** [comment]
IMO it looks good enough.

**Pull request merged**